backend/app/literature/tests/test_001_reference.py

```diff
@@ -186,6 +188,7 @@ def test_reference_large():
     assert res['cross_references'][0]['curie'] == 'FB:FBrf0221304'
     # cross references in the db?
     xref = db.query(CrossReferenceModel).filter(CrossReferenceModel.curie == "FB:FBrf0221304").one()
+    print("BOB: xref is '{}'".format(xref))
```

Comment:
BOB!

Reply:
I was not too bothered about BOB in the tests as it only is seen if the tests fail. But I will remove it as general if you see BOB in my code then it means I wanted to remove it later. (so if anyone sees BOB in my code it means it was for debugging and I intended to remove it later)